src/popup/components/TokenAmount.vue

```diff
@@ -141,5 +144,13 @@ export default defineComponent({
       @extend %face-sans-18-regular;
     }
   }
+
+  &.align-right {
+    text-align: right;
+
+    .fiat {
+      @extend %face-sans-12-regular;
+    }
```

Comment:
I think it's bad to change the font weight of elements if we are changing the alignment. Those 2 things are not connected.